### ISSUE TITLE
Add progress bar to processing

### ISSUE THREAD

**Issue by michael-sky** [dsaffo/DataComets#6]
After the upload is completed, it would be nice have a progress bar or similar to let the user know how about the progress. Especially with larger log files.

**Comment by dsaffo** [comment]
Agreed, I will work to get that integrated on the client side it is visible on the server side.
